src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -91,38 +94,19 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
   }
 
   const resetOrder = async () => {
-    // TODO: reset fulfillment type: Not yet supported
-    // const result = await setFulfillmentOptionMutation.submitMutation({
-    //   variables: {
-    //     input: {
-    //       id: orderData.internalID,
-    //       fulfillmentOption: null,
-    //     },
-    //   },
-    // })
-    // validateAndExtractOrderResponse(result.setOrderFulfillmentOption?.orderOrError)
-
-    // reset fulfillment details
-    const fulfillmentDetailsResult = await updateOrderMutation.submitMutation({
-      variables: {
-        input: {
-          id: orderData.internalID,
-          shippingName: null,
-          shippingAddressLine1: null,
-          shippingAddressLine2: null,
-          shippingPostalCode: null,
-          shippingCity: null,
-          shippingRegion: null,
-          shippingCountry: null,
-          buyerPhoneNumber: null,
-          buyerPhoneNumberCountryCode: null,
+    const unsetFulfillmentOptions =
+      await unsetFulfillmentOptionMutation.submitMutation({
```

Comment:
Confused here - why do we not need to unset these other values? Maybe totally reasonable, i just don't understand how we hand responsibility back to the legacy shipping route. It probably needs to save things again too, right?